### ISSUE TITLE
Export to the info format

### ISSUE THREAD

**Issue by aadcg** [progit/progit2#1425]
Thank you for this amazing book.

I'm wondering if it's possible to export it to the info format. Lots of people would be happy to be able to read it from Emacs.

**Comment by jnavila** [comment]
Good question! I am not aware of any asciidoc→ info converter, but maybe asciidoc→docbook→info would be a path.

Anyway, does info support pictures? The book is full of picture and stripping them would not be an option.

If you want to dig this path, we'd be happy to include it into our artefact generation system. Keep us updated on your progress.

**Comment by aadcg** [comment]
> asciidoc→docbook→info would be a path

Yes, seems so. I found [this](http://docbook2x.sourceforge.net/latest/doc/docbook2texi.html).

> does info support pictures?

I'm running Emacs 26.3 (on GTK) and the manual for Guix does include images. 

Could be give me an overview of what needs to be done? I'm not experienced.

In the meantime, Emacs users can benefit from the extension pdf-tools to read PDFs.
